packages/fxa-auth-server/lib/senders/index.js

```diff
@@ -9,292 +9,90 @@ const createSms = require('./sms');
       sms: createSms(log, translator, templates, config),
     };
   }
 
   function splitEmails(emails) {
-    const result = emails.reduce(
+    return emails.reduce(
       (result, item) => {
+        const { email } = item;
+
         if (item.isPrimary) {
-          result.to = item.email;
+          result.to = email;
         } else if (item.isVerified) {
-          result.cc.push(item.email);
+          result.cc.push(email);
         }
+
         return result;
       },
       { cc: [] }
     );
-
-    if (!result.to) {
-      throw error.unexpectedError();
-    }
-
-    return result;
   }
 
   const senders = await createSenders();
   const mailer = senders.email;
 
-  senders.email = {
-    sendVerifyCode(emails, account, options) {
-      return mailer.verifyEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        email: account.email,
-        uid: account.uid,
-      });
-    },
-    sendVerifyShortCode(emails, account, options) {
-      // This function differs from `sendVerifyCode` since it sends a code that
-      // is expected to be entered into a input field rather than a link to verify
-      // the account. It is expected to be much shorter than the `emailCode`.
-      return mailer.verifyShortCodeEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        email: account.email,
-        uid: account.uid,
-      });
-    },
-    sendVerifyLoginEmail(emails, account, options) {
-      const { to, cc } = splitEmails(emails);
-
-      return mailer.verifyLoginEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        ccEmails: cc,
-        email: to,
-        uid: account.uid,
-      });
-    },
-    sendVerifyLoginCodeEmail(emails, account, options) {
-      const { to, cc } = splitEmails(emails);
-
-      return mailer.verifyLoginCodeEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        ccEmails: cc,
-        email: to,
-        uid: account.uid,
-      });
-    },
-    sendVerifyPrimaryEmail(emails, account, options) {
-      return mailer.verifyPrimaryEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        email: account.email,
-        uid: account.uid,
-      });
-    },
-    sendVerifySecondaryEmail(emails, account, options) {
-      return mailer.verifySecondaryEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        email: emails[0].email,
-        primaryEmail: account.email,
-        uid: account.uid,
-      });
-    },
-    sendRecoveryCode(emails, account, options) {
-      const { to, cc } = splitEmails(emails);
-
-      return mailer.recoveryEmail({
-        ...options,
-        acceptLanguage: options.acceptLanguage || defaultLanguage,
-        ccEmails: cc,
-        email: to,
-        emailToHashWith: account.email,
```

Comment:
Nice! By removing this level, less likely to miss passing some of the email options.